docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -307,7 +307,7 @@ checking that the kube-dns pod is Running in the output of `kubectl get pods --a
 And once the kube-dns pod is up and running, you can continue by joining your nodes.
 
 If your network is not working or kube-dns is not in the Running state, check
-out the [troubleshooting section](#troubleshooting) below.
+out the [troubleshooting section](/docs/setup/independent/troubleshooting-kubeadm/) below.
```

Comment:
If we still use the wording `troubleshooting section` here, then we should link to the section anchor #troubleshooting in this doc as it was.